ci-operator/config/openshift/cluster-api-provider-ovirt/openshift-cluster-api-provider-ovirt-master.yaml

```diff
@@ -9,7 +9,6 @@ build_root:
     name: release
     namespace: openshift
     tag: golang-1.13
-    canonical_go_repository: github.com/ovirt/cluster-api-provider-ovirt
```

Comment:
Is this expected?
`canonical_go_repository` seems a [valid field](https://github.com/openshift/ci-tools/blob/master/CONFIGURATION.md).